source/ui/Components/Output/Console.qml

```diff
@@ -1,75 +1,73 @@
 import QtQuick 2.0
 import QtQuick 2.3
 import QtQuick.Controls 1.4
+import QtQuick.Controls.Styles 1.4
 
 Item {
     property int outputItemIndex: 2
     property int oldMode: 0
     property var oldAddress: 0
     id: it
 
-
-    Rectangle{
-        anchors.fill: parent
-        color: "black"
-
-        TextArea{
-            id: textarea
-            height: parent.height
-            width: parent.width
-            anchors.top: parent.top
-            anchors.left: parent.left
-
-            wrapMode: Text.Wrap
+    TextArea {
+        id: textarea
+        height: parent.height
```

Comment:
Consider using `anchors` over property bindings as anchors are implemented more efficient in qml. Or just use `anchors.fill: parent`